backends/platform/android/events.cpp

```diff
@@ -1159,19 +1175,26 @@ void OSystem_Android::pushEvent(int type, int arg1, int arg2, int arg3,
 		switch (arg1) {
 		// AMOTION_EVENT_ACTION_MOVE is 2 in NDK (https://developer.android.com/ndk/reference/group/input)
 		case AMOTION_EVENT_ACTION_MOVE:
-			e.mouse = dynamic_cast<AndroidCommonGraphics *>(_graphicsManager)->getMousePosition();
-			e.type = Common::EVENT_MOUSEMOVE;
-
 			// already multiplied by 100
-			e.mouse.x += arg2 * _joystick_scale / _eventScaleX;
-			e.mouse.y += arg3 * _joystick_scale / _eventScaleY;
+			_axisPosX = (int32)arg2 * Common::JOYAXIS_MAX / _eventScaleX;
+			_axisPosY = (int32)arg3 * Common::JOYAXIS_MAX / _eventScaleY;
```

Comment:
It might be cleaner to handle axis scaling in the Java code instead.

Reply:
Do you mean redefine `JOYAXIS_MAX` and `JOYAXIS_MIN` with the same values on the Java side so we can use the values directly here?

Reply:
Sure, that should be okay.

Reply:
@ccawley2011 I just added a commit that moves the axis scaling to the Java code (also rebased onto `master` to get the build working).